invenio_app_ils/ui/backoffice/src/pages/ItemDetails/state/types.js

```diff
@@ -0,0 +1,6 @@
+export const SET_ITEM_FETCH_LOADING = 'SET_ITEM_FETCH_LOADING';
+export const SET_ITEM_FETCH_DETAILS = 'SET_ITEM_FETCH_DETAILS';
+
+export const SET_ITEM_ACTION_LOADING = 'SET_ITEM_ACTION_LOADING';
+export const SET_ITEM_ACTION = 'SET_ITEM_ACTION';
```

Comment:
What about start renaming these things to `ITEM_ACTION_SUCCESS` and use the SET prefix only if we just set a flag that changes the components state like the loading or error state?